vllm/config/vllm.py

```diff
@@ -441,25 +607,24 @@ def __post_init__(self):
             else:
                 self.compilation_config.custom_ops.append("all")
 
+        default_config = OPTIMIZATION_LEVEL_TO_CONFIG[self.optimization_level]
+        self._apply_optimization_level_defaults(default_config)
+        assert self.compilation_config.mode >= CompilationMode.NONE
+        assert self.compilation_config.mode <= CompilationMode.VLLM_COMPILE
+
         # async tp is built on top of sequence parallelism
         # and requires it to be enabled.
         if self.compilation_config.pass_config.enable_async_tp:
             self.compilation_config.pass_config.enable_sequence_parallelism = True
         if self.compilation_config.pass_config.enable_sequence_parallelism:
-            self.compilation_config.custom_ops.append("+rms_norm")
+            if "-rms_norm" in self.compilation_config.custom_ops:
```

Comment:
I'm not familiar with the notation but presumably this should be a `+`?

```suggestion
            if "+rms_norm" in self.compilation_config.custom_ops:
```